aws/jupyter-config.yaml

```diff
@@ -0,0 +1,54 @@
+proxy:
+  secretToken: "e1e761517a8f4c025144e53be60ffc482e6321e26d8a184c2bab289948b8ba13"
```

Comment:
Not a good idea to have secret checked in!

Reply:
Yep, should've checked before committing.